cspell.config.yaml

```
@@ -1,4 +1,4 @@
-version: '0.2'
+version: "0.2"
```

```suggestion
version: '0.2'
```

Maybe I should make ESLint ignore this file...